certs/postgres/setup_test.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/jmoiron/sqlx"
 	"github.com/mainflux/mainflux/certs/postgres"
-	pgClient "github.com/mainflux/mainflux/internal/clients/postgres"
+	pgclient "github.com/mainflux/mainflux/internal/clients/postgres"
 	"github.com/mainflux/mainflux/logger"
 	dockertest "github.com/ory/dockertest/v3"
```

Comment:
doesn't require an alias

Reply:
It does, we have:
`	"github.com/mainflux/mainflux/certs/postgres" `
and
`	pgClient "github.com/mainflux/mainflux/internal/clients/postgres"`